src/gameplayv2/resistance/components/AssassinateComponent.ts

```diff
@@ -0,0 +1,102 @@
+import Game from "../../Game";
+
+export class AssassinateComponent {
+    protected assassin: string;
+    protected isEnabled: boolean;
+    protected role: string = 'Assassin';
```

Comment:
There's no concept of `role` in a component. A role is an entity that is made up of a collection of components!